Bottom: 0b4eba7bbc252fe650dde131d74a87954e056bdc
Top:    4c7da91b61d14db13f493c97414dcfc0bb4ddc23
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-22 22:40:23 +0100

drc: HACK HACK HACK


---

diff --git a/src/Makefile.am b/src/Makefile.am
index 61eacba..8f698f4 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -283,6 +283,7 @@ libgtk_a_CPPFLAGS = -I./hid/gtk
 LIBGTK_SRCS = \
 	dolists.h \
 	hid/hidint.h \
+	hid/gtk/bga-wizard.c \
 	hid/gtk/gtkhid-main.c \
 	hid/gtk/gtkhid.h \
 	hid/gtk/gui.h \
